CHANGELOG.md

```diff
@@ -45,6 +45,9 @@
 
 ##### Bug Fixes
 
+* Rule out a few invalid `@IBInspectable` cases in `valid_ibinspectable`.
```

Comment:
I think it's to force markdown to add a `<br/>`. From the [original spec](https://daringfireball.net/projects/markdown/syntax#p):

> When you do want to insert a `<br />` break tag using Markdown, you end a line with two or more spaces, then type return.